requirements/base.txt

```diff
@@ -1,7 +1,7 @@
 chevron~=0.12
 # 8.1.4 of Click has an issue with the typing breaking the linter - https://github.com/pallets/click/issues/2558
 # Allow click to be greater than 8.1.4 when https://github.com/pallets/click/pull/2565 is released.
-click~=8.0,<8.1.4
+click~=8.1
```

Comment:
Looks like this PR still picked up `8.1.5` for the Windows 3.9 `make pr` job.
```
Requirement already satisfied: click~=8.1 in c:\hostedtoolcache\windows\python\3.9.13\x64\lib\site-packages (from aws-sam-cli==1.92.0) (8.1.5)
```

Thoughts on changing this to
```suggestion
click~=8.1.6
```

Reply:
Is it fixed now since Windows jobs are succeeding?

Reply:
But weird enough, windows reproducible picked existing version 🤔 
https://github.com/aws/aws-sam-cli/actions/runs/5761609268/job/15619767697?pr=5575#step:4:595